### PR TITLE
Add support for start region migration

### PR DESCRIPTION
**Pull request by johnsudaar**
Related to https://github.com/Scalingo/storky/issues/1

### CODE REVIEW

File: events_structs.go [hunk 1]
```diff
@@ -806,6 +807,19 @@ func (ev *EventDeleteAutoscalerType) String() string {
...
+	MigrationID string `json:"migration_id"`
+}
+
+type EventStartRegionMigrationType struct {
+	Event
+	TypeData EventStartRegionMigrationTypeData `json:"type_data"`
+}
+
+func (ev *EventStartRegionMigrationType) String() string {
+	return "Application region migration started"
```
[the remark this answers is not in the document]
Comment (johnsudaar) [reply events_structs.go]: Not easily :/ This is not stored in the event...

[the remark this answers is not in the document]
Comment (Soulou) [reply events_structs.go]: Maybe it should be added then?

[the remark this answers is not in the document]
Comment (johnsudaar) [reply events_structs.go]: Yep

File: events_structs.go [hunk 2]
```diff
@@ -818,6 +819,21 @@ func (ev *EventDeleteAutoscalerType) String() string {
...
+	DstAppName  string `json:"dst_app_name"`
+}
+
+type EventStartRegionMigrationType struct {
+	Event
+	TypeData EventStartRegionMigrationTypeData `json:"type_data"`
+}
+
+func (ev *EventStartRegionMigrationType) String() string {
+	return fmt.Sprintf("Application region migration started to %s/%s", ev.TypeData.Destination, ev.TypeData.DstAppName)
```
Comment (EtienneM) [inline events_structs.go]: It would be great to also include the source of the migration

Comment (johnsudaar) [reply events_structs.go]: You know the source it's the app you asked the timeline for.

Comment (johnsudaar) [reply events_structs.go]: We never include the source of the events in those messages.

Comment (EtienneM) [reply events_structs.go]: It's not necessarily obvious. If I type:

```
scalingo -a biniou timeline
```

What is the source?

Comment (johnsudaar) [reply events_structs.go]: biniou ?

Comment (EtienneM) [reply events_structs.go]: No, I mean the source region. It's probably `agora-fr1` for instance.

Comment (EtienneM) [reply events_structs.go]: The perfect message for me would be something like:

```
Application region migration from agora-fr1 to osc-fr1/biniou started

Comment (johnsudaar) [reply events_structs.go]: Oh ... It's not that easy. I'll check how to do that.

**Pull request merged**